cnames_active.js

```diff
@@ -571,6 +571,7 @@ var cnames_active = {
   "clickforhelp": "clickforhelp.netlify.app",
   "clickhouse": "depyronick.github.io/clickhouse-client",
   "clientworker": "chenyfan.github.io/ClientWorker",
+  "clipx": "clipx.dzr.app",
```

Comment:
```suggestion
  "clipx": "cname.vercel-dns.com", // noCF
```
👋 It looks like this is a Vercel site, so I imagine you want to actually target Vercel directly here, with the Cloudflare proxy disabled, and then add the requested JS.org subdomain as a custom domain to your Vercel project.